eng/official.yml

```diff
@@ -0,0 +1,156 @@
+# This Yaml Document has been converted by ESAI Yaml Pipeline Conversion Tool.
+# Please make sure to check all the converted content, it is your team's responsibility to make sure that the pipeline is still valid and functions as expected.
+# This pipeline will be extended to the OneESPT template
+# If you are not using the E+D shared hosted pool with windows-2022, replace the pool section with your hosted pool, os, and image name. If you are using a Linux image, you must specify an additional windows image for SDL: https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/1es-pipeline-templates/features/sdlanalysis/overview#how-to-specify-a-windows-pool-for-the-sdl-source-analysis-stage
+trigger:
+  batch: true
+  branches:
+    include:
+    - main
+    - release/*
+    - mono-*
+pr:
+  branches:
+    include:
+    - main
+    - release/*
+    - mono-*
+    - feature/*
+variables:
+- name: officialBuild
+  value: ${{ and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}
+- name: _BuildConfig
+  value: Release
+- name: _BuildArgs
+  value: ''
+- name: _DotNetArtifactsCategory
+  value: .NETCore
+- ${{ if eq(variables.officialBuild, 'true') }}:
+  - name: _BuildArgs
+    value: ${{ format('{0} /p:OfficialBuildId=$(Build.BuildNumber) /p:Test=false /p:IntegrationTest=false', variables['_BuildArgs']) }}
+  - group: DotNet-HelixApi-Access
+  - name: Codeql.Enabled
+    value: True
+  - name: Codeql.TSAEnabled
+    value: True
+resources:
+  repositories:
+  - repository: 1ESPipelineTemplates
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
+  parameters:
+    pool:
+      name: Azure-Pipelines-1ESPT-ExDShared
+      image: windows-2022
+      os: windows
+    customBuildTags:
+    - ES365AIMigrationTooling
+    stages:
+    - stage: build
+      displayName: Build
+      jobs:
+      - template: /eng/common/templates/jobs/jobs.yml@self
+        parameters:
+          enableTelemetry: true
+          helixRepo: dotnet/linker
+          enablePublishUsingPipelines: true
+          enablePublishBuildArtifacts: true
+          ${{ if eq(variables.officialBuild, 'false') }}:
+            enablePublishTestResults: true
+            testResultsFormat: vstest
```

Comment:
we can probably clean up the pipeline to not run tests etc

Reply:
True